src/Identity.Governance/Identity.Governance/custom/Get-MgEntitlementManagementAccessPackage.ps1

```diff
@@ -0,0 +1,210 @@
+    ${HttpPipelineAppend},
+
+    [Parameter(DontShow)]
+    [ValidateNotNull()]
+    [Microsoft.Graph.PowerShell.Category('Runtime')]
+    [Microsoft.Graph.PowerShell.Runtime.SendAsyncStep[]]
+    # SendAsync Pipeline Steps to be prepended to the front of the pipeline
+    ${HttpPipelinePrepend},
+
+    [Parameter(DontShow)]
+    [Microsoft.Graph.PowerShell.Category('Runtime')]
+    [System.Uri]
+    # The URI for the proxy server to use
+    ${Proxy},
+
+    [Parameter(DontShow)]
+    [ValidateNotNull()]
+    [Microsoft.Graph.PowerShell.Category('Runtime')]
+    [System.Management.Automation.PSCredential]
+    # Credentials for a proxy server to use for the remote call
+    ${ProxyCredential},
+
+    [Parameter(DontShow)]
+    [Microsoft.Graph.PowerShell.Category('Runtime')]
+    [System.Management.Automation.SwitchParameter]
+    # Use the default credentials for the proxy
+    ${ProxyUseDefaultCredentials},
+
+    [Parameter(ParameterSetName='ListAll')]
+    [Parameter(ParameterSetName='DisplayNameEq')]
+    [Parameter(ParameterSetName='DisplayNameContains')]
+    [Parameter(ParameterSetName='CatalogId')]
+    [Microsoft.Graph.PowerShell.Category('Runtime')]
+    [System.Management.Automation.SwitchParameter]
+    # List all pages.
+    ${All}
+)
+
+begin {
+    try {
+        $outBuffer = $null
+        if ($PSBoundParameters.TryGetValue('OutBuffer', [ref]$outBuffer)) {
+            $PSBoundParameters['OutBuffer'] = 1
+        }
+        $parameterSet = $PSCmdlet.ParameterSetName
+
+        if ($parameterSet -eq "DisplayNameEq") {
+            
+            $Filter = "displayName eq '{0}'" -f $DisplayNameEq
+            $PSBoundParameters['Filter'] = $Filter
+            $null = $PSBoundParameters.Remove('DisplayNameEq')
+        
+        } elseif ($parameterSet -eq "DisplayNameContains") {
+            
+            $Filter = "contains(tolower(displayName), '{0}')" -f $DisplayNameContains
+            $PSBoundParameters['Filter'] = $Filter
+            $null = $PSBoundParameters.Remove('DisplayNameContains')
+        } elseif ($parameterSet -eq "CatalogId") {
+
+            $Filter = "accessPackageCatalog/Id eq '{0}'" -f $CatalogId
+            $PSBoundParameters['Filter'] = $Filter
+            $null = $PSBoundParameters.Remove('CatalogId')
+        }
+        
+        if ($PSBoundParameters.ContainsKey('Top') -or $PSBoundParameters.ContainsKey('All')) {
+
+        } else {
+            $PSBoundParameters['All'] = $true
+        }
+
+        $mappedCmdList = 'Microsoft.Graph.Identity.Governance.private\Get-MgEntitlementManagementAccessPackage_List';
+        $subWrappedCmd = $ExecutionContext.InvokeCommand.GetCommand(($mappedCmdList), [System.Management.Automation.CommandTypes]::Cmdlet)
+        $subScriptCmd = {& $subWrappedCmd @PSBoundParameters}
+        $steppablePipeline = $subScriptCmd.GetSteppablePipeline($MyInvocation.CommandOrigin)
+        $steppablePipeline.Begin($PSCmdlet)
```

Comment:
Let's avoid creating a steppable pipeline on our own, but instead let AutoREST handle it. We only need to pass the boundParameters to the command we are calling as shown below. The code generator will handle the creation and lifetime management of the steppable pipeline.
```suggestion
        Microsoft.Graph.Identity.Governance.private\Get-MgEntitlementManagementAccessPackage_List @PSBoundParameters
```

Reply:
Updated as suggested across this PR.